packages/contractkit/src/wrappers/Exchange.test.ts

```diff
@@ -0,0 +1,55 @@
+import { newKitFromWeb3 } from '../kit'
+import { testWithGanache } from '../test-utils/ganache-test'
+import { ExchangeWrapper } from './Exchange'
+
+/*
+TEST NOTES:
+- In migrations: The only account that has cUSD is accounts[0]
+*/
+
+testWithGanache('Exchange Wrapper', (web3) => {
+  const ONE = web3.utils.toWei('1', 'ether')
+
+  const kit = newKitFromWeb3(web3)
```

Comment:
Question, this might apply to other tests here as well: since these are kit unit tests, would it be worth it to be creating a new one in a `beforeEach` for each unit test? Maybe not, not sure myself, but something to consider.

Reply:
tbh, i didn't see the need here; and felt more simpler that using a beforeEach